import.py

```diff
@@ -7,1194 +8,22 @@
-    "media/libopus/silk/float/apply_sine_window_FLP.c",
-    "media/libopus/silk/float/autocorrelation_FLP.c",
-    "media/libopus/silk/float/burg_modified_FLP.c",
-    "media/libopus/silk/float/bwexpander_FLP.c",
-    "media/libopus/silk/float/corrMatrix_FLP.c",
-    "media/libopus/silk/float/encode_frame_FLP.c",
-    "media/libopus/silk/float/energy_FLP.c",
-    "media/libopus/silk/float/find_LPC_FLP.c",
-    "media/libopus/silk/float/find_LTP_FLP.c",
-    "media/libopus/silk/float/find_pitch_lags_FLP.c",
-    "media/libopus/silk/float/find_pred_coefs_FLP.c",
-    "media/libopus/silk/float/inner_product_FLP.c",
-    "media/libopus/silk/float/k2a_FLP.c",
-    "media/libopus/silk/float/LPC_analysis_filter_FLP.c",
-    "media/libopus/silk/float/LPC_inv_pred_gain_FLP.c",
-    "media/libopus/silk/float/LTP_analysis_filter_FLP.c",
-    "media/libopus/silk/float/LTP_scale_ctrl_FLP.c",
-    "media/libopus/silk/float/noise_shape_analysis_FLP.c",
-    "media/libopus/silk/float/pitch_analysis_core_FLP.c",
-    "media/libopus/silk/float/process_gains_FLP.c",
-    "media/libopus/silk/float/regularize_correlations_FLP.c",
-    "media/libopus/silk/float/residual_energy_FLP.c",
-    "media/libopus/silk/float/scale_copy_vector_FLP.c",
-    "media/libopus/silk/float/scale_vector_FLP.c",
-    "media/libopus/silk/float/schur_FLP.c",
-    "media/libopus/silk/float/sort_FLP.c",
-    "media/libopus/silk/float/warped_autocorrelation_FLP.c",
-    "media/libopus/silk/float/wrappers_FLP.c",
-    "media/libopus/silk/gain_quant.c",
-    "media/libopus/silk/HP_variable_cutoff.c",
-    "media/libopus/silk/init_decoder.c",
-    "media/libopus/silk/init_encoder.c",
-    "media/libopus/silk/inner_prod_aligned.c",
-    "media/libopus/silk/interpolate.c",
-    "media/libopus/silk/lin2log.c",
-    "media/libopus/silk/log2lin.c",
-    "media/libopus/silk/LP_variable_cutoff.c",
-    "media/libopus/silk/LPC_analysis_filter.c",
-    "media/libopus/silk/LPC_fit.c",
-    "media/libopus/silk/LPC_inv_pred_gain.c",
-    "media/libopus/silk/NLSF2A.c",
-    "media/libopus/silk/NLSF_decode.c",
-    "media/libopus/silk/NLSF_del_dec_quant.c",
-    "media/libopus/silk/NLSF_encode.c",
-    "media/libopus/silk/NLSF_stabilize.c",
-    "media/libopus/silk/NLSF_unpack.c",
-    "media/libopus/silk/NLSF_VQ.c",
-    "media/libopus/silk/NLSF_VQ_weights_laroia.c",
-    "media/libopus/silk/NSQ.c",
-    "media/libopus/silk/NSQ_del_dec.c",
-    "media/libopus/silk/pitch_est_tables.c",
-    "media/libopus/silk/PLC.c",
-    "media/libopus/silk/process_NLSFs.c",
-    "media/libopus/silk/quant_LTP_gains.c",
-    "media/libopus/silk/resampler.c",
-    "media/libopus/silk/resampler_down2.c",
-    "media/libopus/silk/resampler_down2_3.c",
-    "media/libopus/silk/resampler_private_AR2.c",
-    "media/libopus/silk/resampler_private_down_FIR.c",
-    "media/libopus/silk/resampler_private_IIR_FIR.c",
-    "media/libopus/silk/resampler_private_up2_HQ.c",
-    "media/libopus/silk/resampler_rom.c",
-    "media/libopu
```

Comment:
The f.close() should be automatically done when the "with" scope exits. If we're doing the same with with a "with" block elsewhere in this file, we should not do that too.